bdns/dns.go

```diff
@@ -485,15 +485,16 @@ func (dnsClient *DNSClientImpl) LookupHost(ctx context.Context, hostname string)
 
 // LookupCAA sends a DNS query to find all CAA records associated with
 // the provided hostname.
-func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string) ([]*dns.CAA, error) {
+func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string) ([]*dns.CAA, string, error) {
 	dnsType := dns.TypeCAA
+	var response string
```

Comment:
Rather than pre-declaring this, just explicitly return `nil` on lines 493 and 497.